bridges/snowbridge/parachain/pallets/ethereum-beacon-client/README.md

```diff
@@ -0,0 +1,3 @@
+# Ethereum Beacon Client
+
+The Ethereum Beacon Client is an on-chain light client that tracks Ethereum consensus using the beacon chain.
```

Comment:
The README is not linked in the `Cargo.toml`.  

I think at some point we tried to auto-generate the README files from the `lib.rs` docs... but dont think that is currently the case.

Reply:
I read the [cargo docs](https://doc.rust-lang.org/cargo/reference/manifest.html#the-readme-field) and it states that if there is a README.md file in the package root, it will be used:

> If no value is specified for this field, and a file named README.md, README.txt or README exists in the package root, then the name of that file will be used.

Another reason why I didn't link it explicitly is because it doesn't seem to be done consistently in the polkadot-sdk either.

Let me know if you think it's better to link the readme in the cargo file explicitly.